llvm/lib/Target/NVPTX/NVPTXISelDAGToDAG.cpp

```diff
@@ -30,6 +30,10 @@ using namespace llvm;
 #define DEBUG_TYPE "nvptx-isel"
 #define PASS_NAME "NVPTX DAG->DAG Pattern Instruction Selection"
 
+static cl::opt<bool>
+    DisableRsqrtOpt("nvptx-disable-rsqrt-opt", cl::init(false), cl::Hidden,
```

Comment:
Also, a nit on the option name. Disabling a "disable-something" knob always strikes me as odd.  Can we make it a positive control? E.g. `-nvptx-approx-rsqrt` ? 

Naming is hard. :-/

Reply:
hmm, what about `-nvptx-rsqrt-approx-opt` or `-nvptx-rsqrt-approx-folding`? I think it would be good to make clear that this controls optimizing `sqrt` to `rsqrt` not whether `rsqrt` will be emitted at all.

Reply:
Given that it's a hidden option, I don't have particularly strong feelings about the name. `-nvptx-rsqrt-approx-opt` works for me.

Reply:
Okay, I've updated the MR